raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -60,31 +61,85 @@ library ChannelManagerLibrary {
     /// @return (address) the address of the NettingChannelContract.
     function newChannel(
         Data storage self,
+        address caller_address,
         address partner,
         uint settle_timeout)
         returns (address)
     {
         address channel_address;
-        uint i;
+        bool has_channel;
+        uint i_index;
```

Comment:
This is extremely verbose. Consider using simple `i` and `j` or if you really want to go with a verbose version then go for something that's useful. I would suggest `caller_index` and `partner_index` in that case. Still I think it's not necessary.

But if you do that, then please also make the same change in the `getChannelWith()` function above